app/helpers/request.js

```diff
@@ -0,0 +1,19 @@
+const axios = require('axios');
+const { errors } = require('../errors');
+
+const buildRequest = ({ url, path = '', method, headers, data, params }) => ({
```

Comment:
Do you remember what we spoke about at the meeting?

Reply:
Of course, actually, I've changed param names and already commited in future next card in another branch, but no problem, already changed here too! :100: 

- Changed `url` param name to `baseUrl` for better understanding. 
- Kept `path` param the same name based on Swagger Doc I sent to you.

If you have more observations or you are not agree, I'd appreciate letting me knowing it.